src/main/resources/static/account.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<style>
+  #output {
+    width: 35px;
+    height: auto;
+  }
+
+  button {
+    margin-top: 15px;
+  }
+</style>
+
+<body>
+<h1>Account info</h1>
+<div id="loggedIn"></div>
+<form>
+    <p>Write your username</p><input id="Username">
+    <p>Select avatar</p>
+    <main>
+        <input type="file" accept="image/*" onchange="encodeImageFileAsURL(this)">
+        <img id="output" />
+    </main>
+    <button id="confirm">Confirm changes</button>
+</form>
+<script>
```

Comment:
JavaScript code on this page could be improved, but let's do it in scope of GM-32

Reply:
ok!